twitter/src/com/treode/twitter/server/handler/package.scala

```diff
@@ -30,6 +35,20 @@ package object handler {
       rsp
     }
 
+    def apply (req: Request, notifications: Notification [Unit]): Response = {
+      val rsp = req.response
+      notifications match {
+        case errors @ Errors (_) =>
+          val mapper = new ObjectMapper() with ScalaObjectMapper
```

Comment:
Making a mapper is expensive, and a mapper is a fairly static thing. There's one already defined at `com.treode.twitter.finagle.http.mapper` that you should use.